packages/core/src/providers/Providers.tsx

```diff
@@ -29,3 +33,23 @@ function getLibrary(provider: any): Web3Provider {
   library.pollingInterval = 15000
   return library
 }
+
+const BACKGROUND_CHAIN_URLS = {
+  1: 'https://mainnet.infura.io/v3/62687d1a985d4508b2b7a24827551934',
```

Comment:
eee.... `id` in code?
Let's extract it to separate configuration file and use provider that by default not requires id.
Ethers have provider that work out of the box (EtherscanProvider I think).

Also:
Let's fill in urls for all test networks by default